adding increment ID (#480)
infinum/eightshift-forms: CHANGELOG.md

This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a CHANGELOG](https://keepachangelog.com/).

## [5.8.0]

### Added

> - `increment ID` option for entries so you can us them to track success submissions or use it a order number.

Reviewer: ```suggestion
- `increment ID` option for entries so you can use them to track successful submissions or use it as an order number.
```

> - `increment ID` option for entries so you can us them to track success submissions or use it a order number.

Reviewer 2: ```suggestion
- `increment ID` option for entries so you can us them to track success submissions or use it as an order number.
```

- `incrementId` key is now available in the email response tags.

### Changed

> - `entries` listing now has new listing page with better UX and table layout.

Reviewer 2: ```suggestion
- `entries` listing now has the new listing page with better UX and table layout.
```
